src/NotesApplication/Services/NotesMetaDataService.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace ACMTTU.NoteSharing.Platform.NotesApplication.Services {
+    public class NotesMetaDataService {
+        private ConnectionService connectionService;
+
+        /// <summary>
+        /// Consumes the Connection Service so that I can actually use the database and
+        /// the storage blob client
+        /// </summary>
+        /// <param name="connectionService">Service that connects to the database</param>
+        public NotesMetaDataService (ConnectionService connectionService) {
+            this.connectionService = connectionService;
+        }
+
+        /// <summary>
+        /// I can do anything here!!!!
+        /// </summary>
+        public void DoStuff () {
```

Comment:
So this is called from within the `NotesController` class and it seems that the `NotesMetaDataService` is a lightweight wrapper around the `ConnectionService`. Is this the case so we can inject it as a dependence into the `NotesController` class?